frontend/views/containers/proposals/RuleRemoveMember.vue

```diff
@@ -1,19 +1,75 @@
 <template lang="pug">
 proposal-template(
-  :subTitle="L('Change Rule to Remove Member')"
-  :onSubmit='handleSubmit'
-) I step to change rule of remove member it's on the way
+  title= 'L("Change Rule to Remove Member")'
+  footer='L("According to your voting rules, 8 out of 10 members will have to agree with this.")'
+  :disabled='$v.form.$invalid || ($v.steps[config.steps[currentStep]] && $v.steps[config.steps[currentStep]].$invalid)'
+  :maxSteps='config.steps.length'
+  :currentStep.sync='currentStep'
+  @submit='submit'
+)
+
+  .field(v-if='currentStep === 0' key='0')
+    label.label I step to change rule of remove member it's on the way
+
+  .field(v-if='currentStep === 1' key='1')
+    label.label
+      i18n Why are you proposing this change?
+
+    textarea.textarea(
+      name='changeReason'
+      ref='purpose'
+      :placeholder='L("The reason why I\' propositiong this change is...")'
```

Comment:
Now that the text is double-quoted, do you still need to escape the apostrophe here? Also, the `m` is missing after it.